src/main/java/tests/DeleteUserTest.java

```diff
@@ -0,0 +1,50 @@
+package tests;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.Test;
+import rest.ApplicationClient;
+import rest.User;
+import utils.ApiUtils;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class DeleteUserTest extends BaseTest {
+    @Test
+    public void checkDeleteUser() {
+        List<String> availableZipCodes = ApplicationClient.getZipCodes();
+        femaleUser.setZipCode(availableZipCodes.get(0));
+        ApplicationClient.createUser(ApiUtils.fromObjectToJson(femaleUser));
+        ApplicationClient.deleteUser(ApiUtils.fromObjectToJson(femaleUser));
+        List<User> users = ApplicationClient.getUsers();
+        assertFalse(users.contains(users), "User not deleted");
+    }
+
+    @Test
+    public void checkDeleteUserWithRequiredFields() {
+        List<String> availableZipCodes = ApplicationClient.getZipCodes();
+        femaleUser.setZipCode(availableZipCodes.get(0));
+        ApplicationClient.createUser(ApiUtils.fromObjectToJson(femaleUser));
+        femaleUser.setAge(0);
+
+
```

Comment:
It is better logically separate by one line but not two

Reply:
fixed